doc/specs/Keyboard-Selection.md

```diff
@@ -74,6 +112,7 @@ Thanks to Keybinding Args, there will only be 4 new commands that need to be add
 |                       | `Enum direction { up, down, left, right}`                     | The direction the selection will be moved in. |
 |                       | `Enum expansionMode { cell, word, line, viewport, buffer }`   | The context for which to move the selection anchor to. (defaults to `cell`)
 | `selectEntireBuffer`  | | Select the entire text buffer.
+| `toggleMarkMode`      | | Enter or exit mark mode. This allows you to create an entire selection using only the keyboard. |

```

Comment:
wait is the default `toggle` or `hold`?

Reply:
(I'll update the spec after we've had some of these conversations)
Even though Mark Mode would normally be 'hold', I personally like the idea of 'toggle' more. I think we should make it 'toggle'.